src/main/java/jenkins/plugins/office365connector/model/PotentialAction.java

```diff
@@ -82,7 +87,7 @@ public void setType(String type)
     @Override
     public String toString()
     {
-        String temp = "{\"@context\": \"http://schema.org\"," +"\"@type\": \"ViewAction\",\"name\": \""+name+"\", \"target\": [\"" + target + "\"]}";
+        String temp = "{\"@context\": \""+context+"\"," +"\"@type\": \""+type+"\",\"name\": \""+name+"\", \"target\": [\"" + target + "\"]}";
```

Comment:
toSTring that produces JSON format looks strange...

Reply:
Don't know why it's there but I felt I had to either delete it or correct it. I chose the latter.

Reply:
Now it's gone